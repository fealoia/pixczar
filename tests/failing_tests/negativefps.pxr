Void main() {
	Pix tri = new Pix();
  	tri.makeTriangle(200,[200,200,200]);
  	Pix rect = new Pix();
	rect.makeRectangle(100,200,[200,200,200]);
	Placement placed = new Placement(tri,100,300,3,5);
	Placement placed1 = new Placement(rect,200,300,3,5);
	Placement placed2 = new Placement(tri,300,300,3,5);
	Frame[] frames = new Frame[2];
	frames[0] = new Frame();
	frames[1] = new Frame();
	frames[2] = new Frame();
	frames[0].addPlacement(placed);
	frames[1].addPlacement(placed1);
	frames[2].addPlacement(placed2);
	print(render(frames,-1, 800, 600)); 
	//-1 fps
}