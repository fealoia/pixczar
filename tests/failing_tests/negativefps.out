Fatal error: exception Failure("Illegal index")
